src/PensionsRegulatorApi/Controllers/PensionsRegulatorController.cs

```diff
@@ -48,5 +48,33 @@ public async Task<ActionResult<IEnumerable<Organisation>>> Get([FromQuery] strin
                 throw;
             }
         }
+
+        /// <summary>
+        /// Gets the organisations from the pensions regulator for a given PAYE reference
+        /// </summary>
+        /// <param name="payeRef">The PAYE reference from which to get matching organisations from the pensions regulator</param>
+        /// <returns>The organisations for the given PAYE reference from the pensions regulator</returns>
+        /// <response code="200">Success</response>
+        /// <response code="401">The client is not authorized to access this endpoint</response>
+        /// <response code="500">Internal server error</response>
+        [ProducesResponseType(200)]
+        [ProducesResponseType(401)]
+        [ProducesResponseType(500)]
+        [HttpGet("organisations/{aorn}")]
+        public async Task<ActionResult<IEnumerable<Organisation>>> Get(
+            [FromQuery] string payeRef,
+            [FromRoute] string aorn)
```

Comment:
Not a fan of one being route param and one being query param. Makes it look like there is a logic/business reason other than the / decoding issue. Assume this was something you guys have discussed and agreed in the office though in which case I will pipe down.

Reply:
@tom-gough  this is as a result of the discussion we had around the fact that if both are query params then we end up with one method and switching based on null/empty checks on the aorn param.

This 'hybrid' route/query approach was so we could have two distinct methods.

We can comment to explain though.